var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -11,16 +11,16 @@ class Ginkgo(CMakePackage, CudaPackage):
     """High-performance linear algebra library for manycore systems,
     with a focus on sparse solution of linear systems."""
 
-    homepage = "https://github.com/ginkgo-project/ginkgo"
+    homepage = "https://ginkgo-project.github.io/"
     url      = "https://github.com/ginkgo-project/ginkgo.git"
     git      = "https://github.com/ginkgo-project/ginkgo.git"
 
     version('develop', branch='develop')
+    version('latest', branch='master')
```

Comment:
Please don’t call it latest, but ‘master’ instead, version comparison will soon work fine for this case as well.

Reply:
Alright, I put `master` here.